src/pages/edge/edge-network/tutorials/send-datastream-overrides.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Sending datastream overrides to Edge Network
```

Comment:
can we rename this to `Datastream config overrides using sendEvent` for consistency with the other guide?

Reply:
Actually - maybe can we change the other one instead? So change it to "Sending configuration overrides using rules" for the other one and make this one "Sending configuration overrides using sendEvent"?